### PR TITLE
DROTH-827 and Droth 884 bearing needs to be taken into account when adjusting bus stops

### PR DESCRIPTION
**Pull request by MAC-CGI**

### CODE REVIEW

File: digiroad2-oracle/sql/insert_test_fixture.sql [hunk 1]
```diff
@@ -5,4 +5,4 @@ Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_I
 Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (29,null,2,109,109,1140018963,1021227);
 Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (36,null,3,113,113,388554364,6488445);
 Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (37,null,3,113,113,388554364,6488445);
-Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (38,null,3,113,113,388554364,6488445);
+Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (38,null,3,113,113,388554364,6488445);
```
Comment (joanacoelho) [inline digiroad2-oracle/sql/insert_test_fixture.sql]: the id for this insert shouldn't ' be ` lrm_position_primary_key_seq.nextval ` ?

**Comment by joanacoelho** [comment]
ok

File: digiroad2-oracle/sql/insert_test_fixture.sql (hunk 1 above)
Comment (MAC-CGI) [reply digiroad2-oracle/sql/insert_test_fixture.sql]: In this case, the code didn't change. Git found some difference, like a space or enter

File: UI/src/model/SelectedMassTransitStop.js [hunk 2]
```diff
@@ -294,6 +294,7 @@
     var switchDirection = function() {
       var validityDirection = validitydirections.switchDirection(get('validityDirection'));
       setProperty('vaikutussuunta', [{ propertyValue: validityDirection }]);
+      currentAsset.payload.linkId = currentAsset.payload.linkId ? currentAsset.payload.linkId : currentAsset.linkId;
```
[the remark this answers is not in the document]
Comment (leirasf) [reply UI/src/model/SelectedMassTransitStop.js]: It's the same thing. In javascript when you have a undefined it's the same as false.

**Pull request merged**